### ISSUE TITLE
0x80103a01 error in PKGj v0.55

### ISSUE THREAD

**Issue by kingmustard** [blastrock/pkgj#385]
I have a Vita with the 3.60 firmware and SD2Vita (with a 128 GB memory card that's practically empty).

When I try to download a game using PKGj, it shows:

Failed to install [game]: SceDownload change_state DC0 failed: 0x80103a01 error in pkgj

There are no active downloads.

Within Settings > Devices > Storage Devices, ux0 is set to Memory Card and uma0 is set to SD2Vita. Are those correct?

I've noticed, at the bottom-right of PKGj, it shows 'Free 246.56 M'. Is that how much storage it thinks there is? If so, that explains why it can't download any games. It appears not to be downloading them to my memory card.

Any ideas?

**Comment by Masamune3210** [comment]
Yes, that is how much space pkgj detects as being free to use on ux0. I don't think there is a way to have it download to any other mountpoint so for some reason it seems to be having issues with the redirection. Can you upload your plugins.txt?

**Comment by SimonAfek** [comment]
As far as I'm aware, Vita games can only run off of ux0. You need to use storageMgr or YAMT or some other equivalent plugin/app to set your SD2Vita up as ux0.

Adrenaline can be set up to run PSP games off of uma0, but native Vita bubbles can only run from ux0.

**Comment by Masamune3210** [comment]
Oh yeah I completely missed that, yeah that is exactly your issue. You have to mount any storage you wish to use for games as ux0: ,the only things that can use uma0 are Atmosphere and some homebrew
